code/model/sudoku/SudokuPuzzle.java

```diff
@@ -0,0 +1,116 @@
+package code.model.sudoku;
+
+import java.util.Stack;
+import code.model.*;
+
+/**
+ * Sudoku puzzle class
+ */
+public class SudokuPuzzle {
+
+    private int[][] numbers;
+    private boolean[][] numberLocks;
+    private int[][] defaultAnswer;
+    private int selectedNumber;
+    private Stack<SudokuMove> moveHistory;
+
+    /**
+     * Default Constructor
+     */
+    public SudokuPuzzle() {
+        this.numbers = new int[9][9];
+        this.numberLocks = new boolean[9][9];
+        this.defaultAnswer = new int[9][9];
+        this.selectedNumber = null;
+        this.moveHistory = new Stack<SudokuMove>();
+    }
+
+    /**
+     * Gets the numbers
+     * 
+     * @return this.numbers
+     */
+    public int[][] getNumber() {
+        return this.numbers;
+    }
```

Comment:
I intended for these to be getting the number at a given position, which is why the class diagram specifies a `row` and `column` in the method header.

Recommend changing the name of the current method to be plural (`getNumbers()`), and making a second method that follows what is in the class diagram.